polaris_vega:
  ip: '10.8.154.159'
  port: 8765
  fps: 60
  debug: False

passive_tools:
  -
    path: '/home/nml/Downloads/8700339.rom'
  -
    path: '/home/nml/Downloads/8700449.rom'